code/processes/wdb.q

```diff
@@ -7,68 +7,68 @@
-
-writedownmode:@[value;`writedownmode;`default];			/- the wdb process can periodically write data to disc and sort at EOD in two ways:
-														/- 1. default 		- 	the data is partitioned by [ partitiontype ]
-														/-						at EOD the data will be sorted and given attributes according to sort.csv before being moved to hdb
-														/- 2. partbyattr	-	the data is partitioned by [ partitiontype ] and the column(s) assigned the parted attributed in sort.csv
-														/-						at EOD the data will be merged from each partiton before being moved to hdb									
-
-mergenumrows:@[value;`mergenumrows;100000];                     /-default number of rows for merge process
-mergenumtab:@[value;`mergenumtab;`quote`trade!10000 50000];     /-specify number of rows per table for merge process
-														
-hdbtypes:@[value;`hdbtypes;`hdb];                               /-list of hdb types to look for and call in hdb reload
-rdbtypes:@[value;`rdbtypes;`rdb];                               /-list of rdb types to look for and call in rdb reload
-gatewaytypes:@[value;`gatewaytypes;`gateway]					/-list of gateway types to inform at reload
-tickerplanttypes:@[value;`tickerplanttypes;`tickerplant];      	/-list of tickerplant types to try and make a connection to
-tpconnsleepintv:@[value;`tpconnsleepintv;10];                   /-number of seconds between attempts to connect to the tp								
-sorttypes:@[value;`sorttypes;`sort];                   		/-list of sort types to look for upon a sort		
-sortslavetypes:@[value;`sortslavetypes;`sortslave];             /-list of sort types to look for upon a sort being called with slave process		
-										
-subtabs:@[value;`subtabs;`]                                     /-list of tables to subscribe for
-subsyms:@[value;`subsyms;`]                                     /-list of syms to subscription to
-upd:@[value;`upd;{insert}]                                      /-value of the upd function
-
-ignorelist:@[value;`ignorelist;`heartbeat`logmsg]               /-list of tables to ignore
-replay:@[value;`replay;1b]                                      /-replay the tickerplant log file
-schema:@[value;`schema;1b]                                      /-retrieve schema from tickerplant
-numrows:@[value;`numrows;100000]                                /-default number of rows 
-savedir:@[value;`savedir;`:temphdb]                   			/-location to save wdb data
-numtab:@[value;`numtab;`quote`trade!10000 50000]                /-specify number of rows per table
-settimer:@[value;`settimer;0D00:00:10]                          /-set timer interval for row check 
-
-partitiontype:@[value;`partitiontype;`date]                     /-set type of partition (defaults to `date)
-gmttime:@[value;`gmttime;1b]	                                /-define whether the process is on gmttime or not
+mode:@[value;`mode;`saveandsort];	                                /- the wdb process can operate in three modes
```

Comment:
... :| ...

Reply:
It's because you've left a tab on this line.